tests/beta/interfaces/test_tape_tf.py

```diff
@@ -785,3 +783,571 @@ def test_sampling(self):
 
         assert res.shape == (2, 10)
         assert isinstance(res, tf.Tensor)
+
+
+@pytest.mark.parametrize(
+    "dev_name,diff_method", [["default.qubit", "finite-diff"], ["default.qubit.tf", "backprop"]]
+)
+class TestQNode:
+    """Same tests as above, but this time via the QNode interface!"""
+
+    def test_execution_no_interface(self, dev_name, diff_method):
+        """Test execution works without an interface"""

```

Comment:
Might be worth extending the description (e.g. execution with `GradientTape`, trainable parameters being checked)